_posts/2020-01-21-snowplow-objc-tracker-1.2.0-released/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Snowplow Obj-C Tracker 1.2.0 released"
+title-short: Snowplow Obj-C Tracker 0.9.0
+tags: [snowplow, objc, tracker]
+author: Alex Benini
+category: Releases
+permalink: /blog/2020/01/21/snowplow-objc-tracker-1.2.0-released/
+discourse: false
+---
+
+We are pleased to announce a new release of the [Snowplow Obj-C Tracker][objc-tracker].
+
+On [Version 1.2.0][1.2.0-tag] we fixed few a rare and annoying bugs, but the big news is the support for watchOS. It has been added with the contribution from [Leo Mehlig][leoasana] who updated the tracker watchOS making it more complete and multi-purposeful.
+
+Read on below for:
+
+1. [watchOS support](#watchos)
+2. [Overriding of the platform parameter](#platform)
+3. [Removed Reachability.swift dependency](#reachability)
+4. [Updates and bug fixes](#updates)
+5. [Documentation](#documentation)
+6. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="watchos">1. watchOS support</h2>
+
+The important news of this version is the support of the watchOS platform on our tracker ([#465][465]). A big thanks goes to [Leo Mehlig][leoasana] who has contributed updating the tracker so it works properly on the Apple Watch.
+Since the day Apple Watch was released, it has firmly kept its leadership position within the smartwatch market. Its success forces developers who are building Apple Watch apps to understand how to best engage an audience that behaves so differently than on a classic smartphone app. The data tracked on the watch can provide a full picture about how the users use the watch, enabling developers and designers to do the hard job of extracting the full power from the watch apps. The watchOS was the missing platform in our tracker. We already support iOS, macOS and tvOS.
+
+The tracker can be instanced and configured like the tracker in the iOS app. It can be done in the ExtensionDelegate of the watch app or in an InterfaceController if it's a single screen app. There are no differencies in the way you can use it.
+
+Note: The current solution is in alpha version.
```

Comment:
This is a bit of a bombshell to drop at the end of this section. Are we sure it is Alpha really and is it labelled as Alpha anywhere else (Github repo, documentation)? What stops this from being a Beta quality release?

Reply:
ah right, yeah it's a bombshell!
Essentially because I forgot to remove it. :)

But technically is true. I have couple of things to check before to add the related Jira tasks.
Also I think we need to clearly declare what is production ready and what is alpha or beta. It's not clear at the moment.

Btw, for now I'm just going to remove this sentence.